src/litdata/constants.py

```diff
@@ -74,6 +74,7 @@
     17: torch.int64,
     18: torch.long,
     19: torch.bool,
+    20: torch.uint16,
```

Comment:
Mind adding a test ?

Reply:
@tchaton, it looks like we already have one that covers all the torch data types:
https://github.com/Lightning-AI/litData/blob/32950dd3ec0dd1472c212de8a19c04fe08e7db7f/tests/streaming/test_serializer.py#L204-L223

Do we still need a separate one?